apps/extension/src/background/index.ts

```diff
@@ -73,44 +64,42 @@ const init = new Promise<void>(async (resolve) => {
   );
   await initShared(sharedWasm);
 
-  const routerId = await getNamadaRouterId(extensionStore);
+  const routerId = await getNamadaRouterId(localStorage);
   const requester = new ExtensionRequester(messenger, routerId);
-  const broadcaster = new ExtensionBroadcaster(connectedTabsStore, requester);
+  const broadcaster = new ExtensionBroadcaster(localStorage, requester);
 
   const vaultService = new VaultService(
-    store,
+    vaultStorage,
     sessionStore,
     cryptoMemory,
     broadcaster
   );
-  const chainsService = new ChainsService(chainStore, broadcaster);
+  const chainsService = new ChainsService(localStorage, broadcaster);
   const sdkService = new SdkService(chainsService);
   const keyRingService = new KeyRingService(
     vaultService,
     sdkService,
     chainsService,
     utilityStore,
-    connectedTabsStore,
-    extensionStore,
+    localStorage,
+    vaultStorage,
     cryptoMemory,
     requester,
     broadcaster
   );
   const ledgerService = new LedgerService(
     keyRingService,
     sdkService,
-    store,
-    connectedTabsStore,
+    vaultStorage,
     txStore,
-    revealedPKStore,
+    revealedPKStorege,
```

Comment:
Nitpick - this should be `revealedPKStorage`

Reply:
Thanks! :)